packages/common/src/validation/index.ts

```diff
@@ -257,6 +257,7 @@ export const indexName = Joi.string()
   .required()
   .pattern(/^[_a-z]+[_a-z0-9]+/i);
 
-export const CollectionIndex = Joi.object()
-  .pattern(Joi.string(), ESortingSchema) // Keys are strings, values must be 'Asc' or 'Desc'
+export const JOI_MONGO_FIELD = Joi.string()
```

Comment:
Limit to `\^[a-zA-Z][a-zA-Z0-9\_]\` would be better.

Reply:
Solved